Blizzard.IO.RabbitMQ.Tests/NetqRabbitPublisherTests.cs

```diff
@@ -0,0 +1,111 @@
+﻿using Blizzard.IO.RabbitMQ.Entities;
+using EasyNetQ;
+using EasyNetQ.Topology;
+using Moq;
+using NUnit.Framework;
+
+namespace Blizzard.IO.RabbitMQ.Tests
+{
+    [TestFixture]
+    class NetqRabbitPublisherTests
+    {
+        private Mock<IBus> _busMock;
+        private Mock<Core.ISerializer<string>> _serializerMock;
```

Comment:
use the aliasing option when you declare a new type like this
`using ISerializer = Blizzard.IO.Core.ISerializer`

Reply:
f